.github/workflows/pytest.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       matrix:
         os: ["ubuntu-latest", "macos-latest", "windows-latest"]
-        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
+        python-version: ["3.10", "3.11", "3.12", "3.13"]
```

Comment:
Dropping support for Python 3.9 (Since the VS Code Python extension debugger no longer supports it) and it will only be a supported Python version for a few more months anyway.